src/__test__/components/data-exploration/generic-gene-table/SelectionActions.test.jsx

```diff
@@ -206,4 +175,33 @@ describe('SelectionIndicator', () => {
 
     expect(component.find('#testExtraOption').length).toEqual(1);
   });
+
+  // uncomment following test when the create cell set modal is openable again
+
+  // test('opens and closes create cell set modal', () => {
+  //   const state = _.cloneDeep(initialState);
+  //   state.genes.selected = ['CEMIP', 'TIMP3'];
+
+  //   const store = mockStore(state);
+  //   const component = mount(
+  //     <Provider store={store}>
+  //       <SelectionActions
+  //         experimentId='test'
+  //         extraOptions={<div id='testExtraOption' />}
+  //       />
+  //     </Provider>,
+  //   );
+
+  //   const button = component.find(Button).at(4);
+  //   button.simulate('click');
+  //   expect(component.find(CreateCellSetModal).length).toEqual(1);
+
+  //   // check if the genes are passed
+  //   expect(component.find(CreateCellSetModal).prop('selectedGenes'))
+  //      .toStrictEqual(['CEMIP', 'TIMP3']);
+
+  //   const closeButton = component.find('.ant-modal-close');
+  //   closeButton.simulate('click');
+  //   expect(component.find(CreateCellSetModal).length).toEqual(0);
+  // });
```

Comment:
Commented tests is not a good idea -- we can easily forget about them and never change them. Better to uncomment them and change them to test for the behaviour that you are committing to master now. Later on, when we change the component fucntionality, we will adapt the tests.